packages/wdio-browserstack-service/src/util.ts

```diff
@@ -430,9 +433,17 @@ export const createAccessibilityTestRun = errorHandler(async function createAcce
         if (response.data.id) {
             process.env.BS_A11Y_TEST_RUN_ID = response.data.id
         }
-
         BStackLogger.debug(`BrowserStack Accessibility Automation Test Run ID: ${response.data.id}`)
 
+        try {
+            if (response.data) {
+                AccessibilityScripts.update(response.data)
+                AccessibilityScripts.store()
+            }
+        } catch {
+            /* Do nothing */
+        }
```

Comment:
Let's not just wrap code into try catch without doing anything about it. If we expect something to throw, we should handle the error otherwise an error will tell us what's wrong and we should not use `try/catch`

```suggestion
        if (response.data) {
            AccessibilityScripts.update(response.data)
            AccessibilityScripts.store()
        }
```

Reply:
@christian-bromann addressed the comments. Please check